dmscripts/notify_suppliers_of_awarded_briefs.py

```diff
@@ -0,0 +1,86 @@
+from datetime import datetime, timedelta
+
+from dmutils.email.exceptions import EmailError
+from dmutils.email.helpers import hash_string
+from dmutils.formats import DATE_FORMAT
+from dmscripts.helpers import env_helpers
+
+
+def create_context_for_brief(stage, brief):
+    return {
+        'brief_title': brief['title'],
+        'brief_link': '{}/{}/opportunities/{}'.format(
+            env_helpers.get_web_url_from_stage(stage),
+            brief['framework']['family'],
+            brief['id']
+        )
+    }
+
+
+def main(
+    data_api_client, mail_client, template_id, stage, logger,
+    dry_run=False, brief_response_ids=None, awarded_at=None
+):
+    failed_brief_responses, brief_responses = [], []
+
+    if brief_response_ids:
```

Comment:
Could we have a comment above this block please to describe what this is trying to do?

Reply:
There is the `# Used for re-running failed email sending` comment below?

Reply:
I basically just mean that the purpose of the entire block is just to populate `brief_responses` and that's not totally clear, but it could be given a comment.

Reply:
functionised 🐳